HOC/MotionWrapperCardDiv.tsx

```diff
@@ -12,13 +12,16 @@ const MotionWrapperCardDiv = ({
   delay,
   duration,
   additionalStyles = '',
+  once = true,
+  translateX,
 }: MotionWrapperCardDivProps) => (
   <motion.div
     variants={fadeIn(direction, type, delay, duration)}
-    viewport={{ once: false, amount: 0.1 }}
+    viewport={{ once, amount: 0.1 }}
     initial="hidden"
     whileInView="show"
-    className={additionalStyles}
+    className={`${additionalStyles} `}
```

Comment:
```suggestion
    className={additionalStyles}
```
If there are no other styles added, there's no need for a template literal